SU2_CFD/src/CMarkerProfileReaderFVM.cpp

```diff
@@ -428,13 +458,72 @@ void CMarkerProfileReaderFVM::WriteMarkerProfileTemplate() {
       /*--- Access the default data for this marker. ---*/
 
       string Marker_Tag = profileTags[iMarker];
-
+      su2double p_total   = config->GetInlet_Ptotal(Marker_Tag);
+      su2double t_total   = config->GetInlet_Ttotal(Marker_Tag);
+      auto flow_dir = config->GetInlet_FlowDir(Marker_Tag);
+      
       /*--- Header information for this marker. ---*/
 
       node_file << "MARKER_TAG= " << Marker_Tag              << endl;
       node_file << "NROW="        << numberOfRowsInProfile[iMarker] << endl;
       node_file << "NCOL="        << nColumns          << endl;
 
+      /*--- start of the header line (names of the columns) --- */
+
+      if (dimension==2)  
+        node_file << "# COORD-X  " << setw(24) << "COORD-Y    " << setw(24);
+      else 
+        node_file << "# COORD-X  " << setw(24) << "COORD-Y    " << setw(24) << "COORD-Z    " << setw(24); 
+
+      INLET_TYPE Kind_Inlet = config->GetKind_Inc_Inlet(Marker_Tag);
+
+      if (compressible) {
+        node_file << "TEMPERATURE" << setw(24) << "PRESSURE   " << setw(24); 
+      }
+      else {
+        switch (Kind_Inlet) {
```

Comment:
What I meant is that this class should not have anything specific to any type of physics.

Reply:
We could construct the entire comment line and a vector (of vectors) of default values outside of the function (so in loadInletProfile in csolver) and pass that to the reader, is that what you mean?

Reply:
I moved the physics from profilereader to csolver.